<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAnalogSet</class>
 <widget class="QDialog" name="CAnalogSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>476</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Analog Set</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>제어값</string>
   </property>
   <widget class="NumEdit" name="editValue">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>27</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>27</y>
      <width>61</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>태그설명</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editTagName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>태그</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editDesc">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>441</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">numedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>CAnalogSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
